src/particles/ImpactXParticleContainer.H

```diff
@@ -107,6 +107,8 @@ namespace impactx
         void
         SetRefParticle (RefPart const refpart);
 
+        RefPart
+        CallRefParticle ();
```

Comment:
Call is good, but we usually simply call this `Get...` for symmetry: `Set`/`Get` :)